crates/ruma-events/src/pdu.rs

```diff
@@ -8,6 +8,8 @@
 //! The stubbed versions of each PDU type remove the `event_id` field (if any)
 //! and the `room_id` field for use in PDU templates.
 
+#![allow(clippy::exhaustive_structs)]
```

Comment:
I said the `EventHash` from further down should probably be non-exhaustive, didn't I? The same also applies to the enum. It's just the PDU structs that can definitely be exhaustive, so it would be better to not do this for the whole module.

Reply:
Yeah you definately did say to make it `non_exhaustive` the wording around this has always confused me it feels like a double negative although it isn't my bad. Fixed.

I added it to the enum too but I'll have to open another PR to enable the lint for enums.